runtime/doc/vim_diff.txt

```diff
@@ -28,7 +28,11 @@ these differences.
   session information.
 
 ==============================================================================
-2. Option defaults					  *nvim-option-defaults*
+2. Defaults					            *nvim-defaults*
+
+- Syntax highlighting is enabled by default
+- Filetype-related plugins and scripts are enabled by default
+  Note: these defaults can be disabled with the "-u NONE" cmdline argument
```

Comment:
Maybe `cmdline` -> `command line`, since `:h cmdline` is regarding ex commands.

A `|-u|` link a at the end might be useful too, like in `starting.txt:401`